build/archives/BUILD.gn

```diff
@@ -74,6 +89,15 @@ if (build_engine_artifacts) {
         destination = "libtessellator$dll"
```

Comment:
Currently I found some file differences when comparing BUILD.gn and GCS artifact. For example, for darwin-x64/artifacts.zip, 
1. the BUILD.gn in cs.opensource suggests the zip file should not contain gen_snapshot, and contain 3 exectuables in total. (https://cs.opensource.google/flutter/engine/+/master:build/archives/BUILD.gn;l=58-69) 
2. the BUILD.gn in this file suggests that the zip file should not contain gen_snapshot, but contain flutter_tester_fractional_translation, with 4 executables in total (https://github.com/flutter/engine/commit/83fef4b32de80f17201e74ae2f8c36b674f0a76c)
3. However, on the latest artifact downloaded from GCS, the zip file extracted contains gen_snapshot, and does not contain flutter_tester_fractional_translation. and has 4 executables in total (https://pantheon.corp.google.com/storage/browser/_details/flutter_infra_release/flutter/ea756c55f29a1f1143a17379c70422d5438ad891/darwin-x64/artifacts.zip;tab=live_object?authuser=0) 
Which one should be the source of truth? Thank you!